ELIPs/ELIP-002.md

```diff
@@ -519,42 +519,77 @@ In this release, when funds are slashed by an AVS, the EigenLayer core contracts
 
 AVSs, taking advantage of Unique Stake, may wish to reward Operator Sets with more granularity in connection to any additional tasks or slashing conditions. This release provides additional functionality in the `RewardsCoordinator` interface for AVSs to specify an Operator Set in rewards submissions to improve the flexibility of incentives and rewards they can provide. The interface will become the sole rewards distribution mechanism with the deprecation of M2 Registration and the `AVSDirectory` at a later date. It is additive with regards to existing [Rewards functionality](https://github.com/eigenfoundation/ELIPs/blob/main/ELIPs/ELIP-001.md#eigenlayer-improvement-proposal-001-rewards-v2).
 
-AVSs specify a set of Strategies for each of their Operator Sets. Reward submissions are verified to only be made to the Strategies in the Operator Set at the time of submission. In addition, reward types are added for making rewards on behalf of an Operator Set as opposed to an AVS. Operators can granularly set their fee on an Operator Set level. If they haven’t, any rewards use the AVS-level fee, or the global default of 10% if nothing has been configured. Programmatic Incentive calculations will be modified to include Operators that have registered to Operator Sets in addition to ones that have registered to AVSs.
+AVSs specify a set of Strategies for each of their Operator Sets. Reward submissions are verified to only be made to the Strategies in the Operator Set for each of the rewards snapshots in the rewards duration, in the sidecar rewards calculation. In addition, reward types are added for making rewards on behalf of an Operator Set as opposed to an AVS. Operators can granularly set their fee on an Operator Set level. If they haven’t, any rewards use the global default of 10% if nothing has been configured. Programmatic Incentive calculations will be modified to include Operators that have registered to Operator Sets in addition to ones that have registered to AVSs. If there are operator set reward submissions with durations prior to the slashing release, those particular reward snapshots will be refunded to the AVS.
```

Comment:
What does ...
> Reward submissions are verified to only be made to the Strategies in the Operator Set for each of the rewards snapshots in the rewards duration, in the sidecar rewards calculation

imply for rewards submitted for strategies not specified in the operator set? Are they ignored? What happens in that unhappy / unspecified case?

Reply:
change sidecar to Eigen Sidecar

Reply:
and capitalize Operator Set in the last sentence

Reply:
If there are strategies submitted that are not specified in the operator set, then they are ignored and the remaining strategies specified are rebalanced based on the specified multipliers.

Reply:
Done in [5430fe3](https://github.com/eigenfoundation/ELIPs/pull/28/commits/5430fe3d02979af320a7cb12fe302afe4b43895d)